lib/leanplum_api/api.rb

```diff
@@ -45,7 +45,9 @@ def track_multi(events: nil, user_attributes: nil, device_attributes: nil, optio
 
     def user_attributes(user_id)
       # Leanplum returns strings instead of booleans
-      Hash[export_user(user_id)['userAttributes'].map { |k, v| [k, v.to_s =~ /\Atrue|false\z/i ? eval(v.downcase) : v] }]
+      export_user(user_id)['userAttributes']
+        .map { |k, v| [k, v.to_s =~ /\A(true|false)\z/i ? eval(v.downcase) : v] }
```

Comment:
i always use `^` and `$` (the old-school unix begin and end zero-width characters), kind of surprised `\A` is uppercase and `\z` is lowercase